docs-kits/kits/behaviour-twin-kit/adoption-view/architecture.md

```diff
@@ -0,0 +1,208 @@
+﻿---
+id: architecture
```

Comment:
This would require a big change of our structure. We prefer to leave it as is. The contents build on each other, so we need less cross-references. This makes the KIT more easy to read.